docs/BREAKINGCHANGES.md

```diff
@@ -6,151 +8,125 @@ TODO
 TODO
 
 
-### Add error on Legacy Credential over non-HTTPS for Web Cmdlets [5402](https://github.com/PowerShell/PowerShell/issues/5402)
-TODO
-
-
 ### Change $OutputEncoding to use iso-8859-1 encoding rather than ASCII [5361](https://github.com/PowerShell/PowerShell/issues/5361)
-TODO
-
+This is marked as a breaking change because the output of the pipeline is not altered as it was previously, anyone relying on ascii encoding (especially for extended ascii sequences) will no longer have that behavior.
```

Comment:
This description isn't correct.  Also, I don't think we need to mention "breaking change" in the description since the doc itself is about breaking changes.  The PR linked to is not valid as we closed that one and decided to use UTF-8 instead of ISO-8859-1.  Correct PR is https://github.com/PowerShell/PowerShell/pull/5369.  I suggest using:

```none
The default encoding of output was previously ASCII (7-bit) which in some cases resulted in incorrect
alteration of the output.  Change is to default to UTF-8 NoBOM which preserves unicode output with an
encoding supported by most tools and operating systems.
```